### PR TITLE
Fixes #24624 - Host Assign Org/Loc radio mismatch input label

### PR DESCRIPTION
**Pull request by ldjebran**
This is needed for automation to handle that radio buttons text without javascript usage.

### CODE REVIEW

**Comment by ldjebran** [comment]
Issue [#24624](https://projects.theforeman.org/issues/24624)

File: app/views/hosts/_taxonomy_import_option.html.erb [hunk 1]
```diff
@@ -1,2 +1,6 @@
+<label class="radio-inline">
   <%= f.radio_button :optimistic_import, "yes" %> <%= _("Fix %s on Mismatch") % taxonomy_type %>
```
Comment (ohadlevy) [inline app/views/hosts/_taxonomy_import_option.html.erb]: could you use https://github.com/theforeman/foreman/blob/develop/app/helpers/form_helper.rb#L99 instead? thanks!

Comment (ldjebran) [reply app/views/hosts/_taxonomy_import_option.html.erb]: @ohadlevy many thanks for the hint, updated

**Comment by ohadlevy** [comment]
[test upgrade]

**Review by ares — APPROVED** [review]
Thanks @ldjebran, fixes the issue and renders fine. The modal deserves a bit more love, but that's out of the scope for this PR. Merging!

**Comment by ares** [comment]
both Foreman and Katello test failures unrelated

**Comment by omaciel** [comment]
@ldjebran ++

**Pull request merged**